### PR TITLE
[ADD] server maintain

### PR DESCRIPTION
**Pull request by noahzaozao**
@elicoidal @seb-elico @xuan139 @victormartinelicocorp Please review and merge this PR.

### CODE REVIEW

File: server_maintain/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline server_maintain/__init__.py:1]: @noahzaozao Can you review this module?
remove/update commented code
Add a README and update short headers/openerp/.

Comment (elicoidal) [reply server_maintain/__init__.py:1]: name of the module should be reviewed as well as it is not obvious to understand the meaning

File: server_maintain/server.py [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+    _description = 'Server User'
+    _rec_name = 'username'
+
+    _columns = {
+        'server_id': fields.many2one(
+            'server.general', 'Server Name', required=True),
+        'username':  fields.char('User Name', size=24, required=True),
+        'password': fields.char('Password'),
+        'type': fields.selection(
+            [('postgres', 'Postgres'), ('openerp', 'OpenERP'),
```
Comment (elicoidal) [inline server_maintain/server.py:137]: s/openerp/Odoo

File: server_maintain/server.py [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+        'linux_users': fields.one2many(
+            'server.user', 'server_id',
+            ondelete='cascade'),
+        'pg_servers': fields.one2many(
+            'server.postgres', 'server_id',
+            ondelete='cascade'),
+        'web_ids': fields.one2many(
+            'maintain.webserver_parameter', 'maintain_id',
+            ondelete='cascade'),
+        # 'oe_module_ids': fields.many2many(
```
Comment (elicoidal) [inline server_maintain/server.py:241]: remove dead code

File: server_maintain/server_maintain.xml [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+                       </group>
+                </page>
+                  <page string="Notes">
+                      <group col="4" colspan="4">
+                       <separator string="Notes" colspan="4"/>
+                             <field name="notes" colspan="4" nolabel="1"/>
+                    </group>
+                    </page>
+                    <page string="Baazar">
+                        <!-- <field name="oe_module_ids"  >
```
Comment (elicoidal) [inline server_maintain/server_maintain.xml:116]: remove dead code

File: server_maintain/server_oe_modules.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from openerp.osv import orm, fields
+from openerp.addons.base_status.base_stage import base_stage
+
+
+class oe_modules(base_stage, orm.Model):
+    _name = 'oe.modules'
+    _description = 'OpenERP Module'
+    _inherit = ['mail.thread', 'ir.needaction_mixin']
+    # _track = {
```
Comment (elicoidal) [inline server_maintain/server_oe_modules.py:10]: remove dead code

**Review by elicoidal — CHANGES_REQUESTED** [review]
cleanse README/short headers/openerp.py and remove dead code.
It is not clear the name and usage of the module

**Comment by elicoidal** [comment]
@seb-elico @Zhouhao12345 do we stil need this code? Can we reuse part of it?
Otherwise, let's close this PR

**Comment by seb-elico** [comment]
@elicoidal Not clear what is the purpose of that module. It looks like a monitoring tool but no idea why we created it (seems like Qiao Lei initiated the code) or if we simply forked/re-used the code of another customer/community project.

Anyway I don't really think this is re-usable but we can always merge it in our addons (once dead code is removed, short headers and Travis are fixed), same way we do for other "old" modules.

**Comment by elicoidal** [comment]
OK: let's scrap it then

**Pull request closed without merging**